### DISCUSSION TITLE
Organization admin gone missing, how to gain admin access for other organization accounts?

### DISCUSSION THREAD

**Discussion by rudy-haryanto** [community/community#36059] [New to GitHub]
We have 2 github accounts to manage the organization in github. But both of them are gone missing. Can we recover the admin prvilege and give it to other organization account?

**Comment by FahdSeddik** [comment]
Hey @rudy-haryanto 
I believe you cannot do that. Please contact support from [here](https://support.github.com/features/organizations). Hope this helps!
